migrator-tool/src/main/scala/za/co/absa/spline/migrator/MigratorCLI.scala

```diff
@@ -57,11 +62,13 @@ class MigratorCLI(migratorTool: MigratorTool) {
         """|A file where a list of failed lineage IDs will be written to.
            |Running migrator with the option '-r' will repeat attempt to migrate lineages from this file."""
           .stripMargin
+        validate (f => if (!f.exists()) success else failure("<failrec> file should not yet exist"))
```

Comment:
I think it should be more like `Failrec file with specified path and name already exists` or just `file already exists`. Say what is wrong, don't say what should (not) be.

Reply:
Make sense, I will change that.